web/util.go

```diff
@@ -138,19 +138,17 @@ func favIcon(w *rest.ResponseWriter, r *rest.Request) {
 	http.ServeFile(
 		w.ResponseWriter,
 		r.Request,
-			staticRoot()+"/ico/zenoss-o.png")
+			staticRoot()+"/ico/favicon.png")
 }
 
 /*
  * Provides content for /licenses.html
  */
 func licenses(w *rest.ResponseWriter, r *rest.Request) {
-	_, filename, _, _ := runtime.Caller(1)
-	licenses := path.Join(path.Dir(filename), "..", "doc", "licenses.html")
 	http.ServeFile(
 		w.ResponseWriter,
 		r.Request,
-		licenses)
+		staticRoot()+"/doc/licenses.html")
 }
```

Comment:
Paths are messed up once this project is packaged and we cannot access files outside the /static directory.